### PR TITLE
SAAP - 197 adding binero Openstack cloud provider documentation

### PR DESCRIPTION
**Pull request by TehreemNisa**
adding binero Openstack cloud provider documentation

### CODE REVIEW

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-205-fddf6482`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-205-7d15353f`

File: content/sre/cloud-provider/binero.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
 # Binero - OpenStack
 
-_TBD_
+A Openstack account is needed to create and manage cluster on Binero Openstack. The following criteria must be met
+
+- Stakater shall be provided access to a separate Binero account or Stakater team can create a Binero Account for cluster provisioning of customer.
+- The Stakater user must have admin access on the created account.
+- Stakater shall share an email address that should have full access to this account.
+- The customer must not use this account for other purposes and only stakater shall be responsible for managing resources created in this account.
```
Comment (rasheedamir) [inline content/sre/cloud-provider/binero.md]: @TehreemNisa this doesn't make any sense; customer might want to provision databases as VM's or create S3 buckets

Comment (TehreemNisa) [reply content/sre/cloud-provider/binero.md]: updated

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-205-ac1e3118`

File: content/sre/cloud-provider/binero.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
 # Binero - OpenStack
 
-_TBD_
+A Openstack account is needed to create and manage cluster on Binero Openstack. The following criteria must be met
+
+- Stakater shall be provided access to a separate Binero account or Stakater team can create a Binero Account for cluster provisioning of customer.
+- The Stakater user must have admin access on the created account.
+- Stakater shall share an email address that should have full access to this account.
```
Comment (rasheedamir) [inline content/sre/cloud-provider/binero.md]: last two statements sound similar

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-205-44c9e360`

File: content/sre/cloud-provider/binero.md (hunk 2 above)
Comment (TehreemNisa) [reply content/sre/cloud-provider/binero.md]: updated , will update for other cloud provider documents aswell

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-205-20aa50a7`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-205-feb93834`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.148`

**Pull request merged**